codegen-server-test/python/build.gradle.kts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+extra["displayName"] = "Smithy :: Rust :: Codegen :: Server :: Python :: Test"
+extra["moduleName"] = "software.amazon.smithy.rust.kotlin.codegen.server.python.test"
+
+tasks["jar"].enabled = false
+
+plugins { id("software.amazon.smithy") }
+
+val smithyVersion: String by project
+val defaultRustFlags: String by project
+val defaultRustDocFlags: String by project
+val properties = PropertyRetriever(rootProject, project)
+
+val pluginName = "rust-server-codegen"
+val workingDirUnderBuildDir = "smithyprojections/python/"
```

Comment:
I'd like the subfolder to be named `codegen-server-test-python`, but I don't know how to do it. Gradle defaults it to the project folder name, which is `python`.

Reply:
The Smithy Gradle plugin has a configuration option for [`outputDirectory`](https://awslabs.github.io/smithy/1.0/guides/building-models/gradle-plugin.html). I haven't used it though, so can't say for certain if it'll do what you're wanting.